portfolio/src/main/webapp/style.css

```diff
@@ -11,6 +11,17 @@
 
 body {
   background-color: #fbfcfc;
+  font-family: 'Roboto', sans-serif;
+}
+
+/* Popup styling */
+#popup {
+  font-size: .8em;
+  visibility: hidden;
```

Comment:
note that toggling `visibility` will still cause space on the page to be allocated for the pop-up before it's shown; if you want the sub-list to be altogether hidden before the pop-up is triggered, you may want to switch to `display` instead

https://stackoverflow.com/questions/133051/what-is-the-difference-between-visibilityhidden-and-displaynone